ios/ExposureNotificationModule.swift

```diff
@@ -144,6 +144,15 @@ public class ExposureNotificationModule: RCTEventEmitter {
            resolve([])
        }
     }
+
+    @objc public func getTestDiagnosisKeys(_ resolve: @escaping RCTPromiseResolveBlock,
```

Comment:
This method is only allowed in development env with the necessary entitlement. This cannot be used when submitting to appstore.